agriculture_shop/data/mail_message.xml

```diff
@@ -6,11 +6,8 @@
         <field name="message_type">email</field>
         <field name="author_id" ref="base.partner_root"/>
         <field name="subtype_id" ref="mail.mt_comment"/>
-        <field name="subject">Knowledge article</field>
+        <field name="subject">🚀 Get started with Odoo Agriculture Shop</field>
         <field name="body" model="knowledge.article" eval="
-            '&lt;br>Hey! Need help?&lt;/br>
-             &lt;br>Checkout this &lt;a href=\''
-             + obj().env.ref('agriculture_shop.knowledge_article_1').article_url.replace(obj().get_base_url(), '')
-             + '\'>article&lt;/a>&lt;/br>'"/>
+        '&lt;span>&#x1F44B; Hi! Follow this &lt;a href=\'/odoo/knowledge/33\'>onboarding guide&lt;/a>. You can find it anytime in the Knowledge app.&lt;/span>'"/>
```

Comment:
33 is valid for you, but surely not for all dbs. You need to use something like `obj().env.ref('agriculture_shop.knowledge_article_1')`